velox/exec/AggregateCompanionAdapter.cpp

```diff
@@ -435,9 +444,11 @@ bool CompanionFunctionsRegistrar::registerExtractFunctionWithSuffix(
             "Signatures whose result types are not resolvable given intermediate types should have been excluded.");
       }
 
+      // TODO: how to get config here
+      core::QueryConfig config({});
       if (auto func = getAggregateFunctionEntry(originalName)) {
         auto fn = func->factory(
-            core::AggregationNode::Step::kFinal, argTypes, resultType);
+            core::AggregationNode::Step::kFinal, argTypes, resultType, config);
```

Comment:
Here is a special case.
It create Aggregate in VectorFunction, should we also add 'core::QueryConfig' to `VectorFunctionFactory`? @mbasmanova

Reply:
Yes, let's do that.

Reply:
Added